foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -728,11 +730,11 @@ msgstr ""
 "Sie können sich mit dem Turris-Omnia-Router am Turris-Projekt beteiligen. "
 "Das Turris-Projekt ist ein gemeinnütziges Forschungsprojekt gegründet von "
 "der Organisation CZ.NIC, welche die .CZ Domäne verwaltet. Falls Sie dem "
-"Projekt beitreten, wird sich Ihr Router als eine Sonde benehmen, welche den "
-"Datenfluss zwischen ihrem Heim und dem Internet analysiert, um arglistige "
-"Datenströme auszufinden. Wenn solcher Datenstrom gefunden ist, dann wird die "
-"Turris-Zentrale notifiziert, welche die Bedrohungsstufe mittels "
-"Datenvergleich festlegen kann. Wenn der Datenstrom als eine Attacke "
+"Projekt beitreten, wird sich Ihr Router als eine Sonde verhalten, welche den "
+"Datenfluss zwischen ihrem Heimnetzwerk und dem Internet analysiert, um "
+"böseartige Datenströme zu finden. Wenn solcher Datenstrom gefunden ist, dann "
+"wird die Turris-Zentrale benachrichtigt, um den verdächtigen Datenstrom mit "
+"den anderen Routern abzugleichen. Wenn der Datenstrom als eine Attacke "
```

Comment:
Here's the problem: They will analyse the data and compare them with other samples but not necessary with data from other routers.
Can there by something like: '.. um den verdächtigen Datenstrom mit Referenzdaten abzugleichen'?